app/controllers/ems_common.rb

```diff
@@ -711,6 +711,7 @@ def set_form_vars
     @edit[:new][:hostname] = @ems.hostname
     @edit[:new][:emstype] = @ems.emstype
     @edit[:amazon_regions] = get_amazon_regions if @ems.kind_of?(ManageIQ::Providers::Amazon::CloudManager)
+    @edit[:google_regions] = get_google_regions if @ems.kind_of?(ManageIQ::Providers::Google::CloudManager)
```

Comment:
So this amazon regions thing is a hack because Amazon does not give regions from an API call.  Does GCE have an API to get regions?  If so, ultimately the EmsRefresh code should just call the API.  But this is good for a first pass (as you don't have connect or any remote querying yet).